azure-pipelines.yml

```diff
@@ -1111,3 +1192,17 @@ jobs:
   - bash: |
       . scripts/ci/install_azdev.sh
       azdev linter --ci-exclusions
+
+- job: VerifySphinxDocumentGenerator
```

Comment:
Do we have document check for extensions CI?

Reply:
Thanks for remind!
I tried it in CI, it failed if I enable
```
test_commands = [sys.argv[1]] if len(sys.argv) >= 2 else default_commands
```

I have already created a bug issue, will fix it in next Sprint S163 https://github.com/Azure/azure-cli/issues/11495